Advent_2a.py

```diff
@@ -0,0 +1,14 @@
+f = open ( 'input.txt' , 'r')
+l = []
+d = []
+l = [line.split() for line in f]
+c = [list(map(int, x)) for x in l]
+
+for i in range(0,16):
+    a = max(c[i])
+    b = min(c[i])
+    diff = a - b
```

Comment:
No need to introduce extra variables, especially when they have such poor names. You can write `diff = max(c[i]) - min(c[i])`